js/components/close-caption-multi-audio-menu/closeCaptionTab.js

```diff
@@ -0,0 +1,39 @@
+var React = require('react');
+var BaseTab = require('../base-components/listWithChoice');
+
+var CloseCaptionTab = React.createClass({
+  componentWillMount: function () {
+    this.header = 'Subtitles';
```

Comment:
This should be defined in constants. Also consider that this might need to be added to the language files otherwise it would show up in English when a different language is used.